gossip/c_block_callbacks.go

```diff
@@ -262,6 +268,14 @@ func consensusCallbackBeginBlockFn(
 							}
 						}
 					}
+					// Skipped txs require original order from events
+					skippedTxs := make([]uint32, 0, len(orderedTxs))
```

Comment:
nit: the upper bound for this list should be the unordered list, right?

Reply:
right. Thanks